src/CoreWCF.Primitives/src/CoreWCF/ServiceAuthenticationManager.cs

```diff
@@ -45,7 +53,7 @@ public override ReadOnlyCollection<IAuthorizationPolicy> Authenticate(ReadOnlyCo
                 authPolicy = authPolicies.AsReadOnly();
             }
 
-            return _wrappedAuthenticationManager.Authenticate(authPolicy, listenUri, ref message);
+            return await _wrappedAuthenticationManager.AuthenticateAsync(authPolicy, listenUri, message);
         }
```

Comment:
When the only await is the last line of the method, you don't need to make the method async. You can directly return the ValueTask from _wrappedAuthenticatonManager

Reply:
i removed the async /await